apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/RevertFinalDecisionDialog.tsx

```diff
@@ -117,101 +105,107 @@ const RevertFinalDecisionDialog = ({
       })
     : intl.formatMessage(commonMessages.notAvailable);
   return (
-    <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
-      <Dialog.Trigger>
-        <Button
-          icon={ArrowUturnLeftIcon}
-          type="button"
-          color="primary"
-          mode="inline"
-        >
-          {intl.formatMessage({
-            defaultMessage: "Revert final decision",
-            id: "AGRCgy",
-            description:
-              "Button label for revert final decision on view pool candidate page",
-          })}
-        </Button>
-      </Dialog.Trigger>
-      <Dialog.Content>
-        <Dialog.Header>
-          {intl.formatMessage({
-            defaultMessage: "Revert final assessment decision",
-            id: "C3YsFy",
-            description:
-              "Subtitle for view revert final decision dialog on view pool candidate page",
-          })}
-        </Dialog.Header>
-        <Dialog.Body>
-          <div data-h2-display="base(grid)" data-h2-gap="base(x1)">
-            <p>
-              {intl.formatMessage(
-                {
-                  defaultMessage:
-                    "Candidate was marked <strong>{decision}</strong> on <strong>{date}</strong>",
-                  id: "jxH8bt",
-                  description:
-                    "Qualified candidate details on revert final decision dialog on view pool candidate page",
-                },
-                {
-                  decision: intl.formatMessage(
-                    isQualified
-                      ? poolCandidateMessages.qualified
-                      : poolCandidateMessages.disqualified,
-                  ),
-                  date: isQualified ? expiryDate : finalDecisionDate,
-                },
+    <div>
```

Comment:
I think we can remove this. It seems to be causing the trigger to wrap.

Reply:
Fixed by [0fbc6f8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10354/commits/0fbc6f8579ce42f9b893c3ee731d294f32d93d74)